cratedb_toolkit/cfr/jobstats.py

```
@@ -88,6 +104,62 @@ def boot(address: DatabaseAddress, report_address: t.Optional[DatabaseAddress] =
+            logger.warning(f"Failed to anonymize statement: {e}")
+    return statement
+
+def deanonymize_statement(statement: str) -> str:
+    """Deanonymize SQL statement using queryanonymizer."""
+    if deanonymize_sql and decoder_dict_path:
+        try:
+            # Load dictionary file each time
+            decoder_dict = {}
+            try:
+                with open(decoder_dict_path, 'r') as f:
+                    decoder_dict = json.load(f)
```

Ruff (0.8.2)

Local variable `decoder_dict` is assigned to but never used

Remove assignment to unused variable `decoder_dict`

(F841)